tests/framework/results/test_manager.py

```diff
@@ -292,20 +294,16 @@ def test_stratified_metrics_initialization():
     STUDENT_HOUSES_LIST = list(STUDENT_HOUSES)
     POWER_LEVELS_STR = [str(lvl) for lvl in POWER_LEVELS]
     assert metrics["house_points"].index.equals(
-        pd.MultiIndex.from_product(
-            [STUDENT_HOUSES_LIST, POWER_LEVELS_STR],
-            names=["student_house", "power_level"],
+        _get_expected_idx(
```

Comment:
This is duplicating the logic of the actual code too much imo. I think you should create the expected MultiIndex directly here and the expected CategoricalIndex below.

Reply:
Kinda hard to move too far away from the code since it's the only way I know how to cram Category dtypes into a MultiIndex. I've tried soemthing else that is at least a bit different. If this is no good then I can just assert that all values are the same and all names are the same